internal/appsec/waf_test.go

```diff
@@ -101,4 +101,34 @@ func TestWAF(t *testing.T) {
 		require.NotNil(t, event)
 		require.True(t, strings.Contains(event.(string), "crs-933-130"))
 	})
+
+	t.Run("obfuscation", func(t *testing.T) {
+		mt := mocktracer.Start()
+		defer mt.Stop()
+		sensitive := "I-m-sensitive-please-dont-expose-me"
+		vulnerable := "1234%20union%20select%20*%20from%20credit_cards"
+
+		// Send malicious request with sensitive information
+		req, err := http.NewRequest("POST", srv.URL+"/?password="+sensitive+vulnerable+sensitive, nil)
```

Comment:
Can you please add a sensitive cookie to the request too please so that we also cover that part:
```go
req.AddCookie(&http.Cookie{Name: "bearer", Value: sensitive+vulnerable2+sensitive})
```
where vulnerable2 will have to be a second attack (because they trigger only once from what I've seen in the system tests)